docs/src/layouts/AppHeader.tsx

```diff
@@ -44,15 +44,15 @@ export default function AppHeader() {
   const t = useTranslate();
 
   React.useEffect(() => {
-    const initialMode = getCookie('paletteMode') || 'system';
+    const initialMode = localStorage.getItem('mui-mode') || 'system';
```

Comment:
We might want to try/catch all the use of `localStorage` added in this PR.

Reply:
Updated.